source/common/config/utility.cc

```diff
@@ -153,6 +153,10 @@ void Utility::validateClusterName(const Upstream::ClusterManager::ClusterSet& pr
 void Utility::checkApiConfigSourceSubscriptionBackingCluster(
     const Upstream::ClusterManager::ClusterSet& primary_clusters,
     const envoy::config::core::v3::ApiConfigSource& api_config_source) {
+  if (api_config_source.api_type() ==
+      envoy::config::core::v3::ApiConfigSource::AGGREGATED_DELTA_GRPC) {
+    return;
+  }
```

Comment:
Is this because the ADS source itself is checked elsewhere? More comments?

Reply:
Comments added.